kaldb/pom.xml

```diff
@@ -21,7 +21,7 @@
         <kafka.version>3.2.0</kafka.version>
         <jackson.version>2.13.3</jackson.version>
         <jackson.databind.version>2.12.7</jackson.databind.version>
-        <lucene.version>8.4.1</lucene.version>
+        <lucene.version>9.0.0</lucene.version>
```

Comment:
I would pick https://central.sonatype.dev/artifact/org.apache.lucene/lucene-core/9.4.0 which is now the latest release

Reply:
9.4 is a bit too new since it's only a few days old. So, upgraded to Lucene 9.3 for now.